crates/ark/src/interface.rs

```diff
@@ -255,6 +255,11 @@ pub struct RMain {
     /// The stack of frames we saw the last time we stopped. Used as a mostly
     /// reliable indication of whether we moved since last time.
     debug_last_stack: Vec<FrameInfoId>,
+
+    /// Current topmost environment on the stack while waiting for input in the
+    /// debugger. This is `Some()` only when R is idle and in a `browser()`
+    /// prompt.
+    pub(crate) debug_env: Option<RObject>,
```

Comment:
I still somewhat strongly prefer removing `pub(crate)` in favor of a helper `fn debug_env(&self) -> Option<&RObject>`. External consumers of `RMain` should not be able to modify this.